### PR TITLE
Fix 5555 status popups

### PR DESCRIPTION
**Pull request by Ka0o0**
As discussed in #5555 this PR removes some obsolete notifications and additionally removes notifications during autosave.
I have also removed the strings from the language file, however, I was not sure if this was correct?

----

- [x] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [x] Manually tested changed features in running JabRef
- [x] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)

### CODE REVIEW

File: src/main/java/org/jabref/gui/shared/SharedDatabaseUIManager.java [hunk 1]
```diff
@@ -51,8 +51,6 @@ public SharedDatabaseUIManager(JabRefFrame jabRefFrame) {
     @Subscribe
     public void listen(ConnectionLostEvent connectionLostEvent) {
 
-        jabRefFrame.getDialogService().notify(Localization.lang("Connection lost."));
```
Comment (koppor) [inline src/main/java/org/jabref/gui/shared/SharedDatabaseUIManager.java]: At least a log should be put here, shouldn't it? - It's about the [shared SQL database](https://help.jabref.org/en/SQLDatabase)

File: src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java [hunk 2]
```diff
@@ -160,9 +159,14 @@ private boolean doSave() {
     }
 
     public boolean save() {
+        return save(false);
+    }
+
+    public boolean save(boolean silent) {
```
Comment (koppor) [inline src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java]: Please, no `boolean` parameters. Please use an Enum. Even `SILENT, LOUD` are as values OK. Boolean should be avoided at (nearly) all costs. See also https://java.by-comparison.com/.

Comment (koppor) [reply src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java]: Please do not use `boolean` as return value. I think, it is never used, is it

Oh, it is also old code. Maybe, you can at least add a JavaDoc comment?

File: src/main/java/org/jabref/gui/shared/SharedDatabaseUIManager.java (hunk 1 above)
Comment (Ka0o0) [reply src/main/java/org/jabref/gui/shared/SharedDatabaseUIManager.java]: Good idea. I have added that to the place, where the event is actually posted.

File: src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java (hunk 2 above)
Comment (Ka0o0) [reply src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java]: Thanks for the suggestion. I agree that enums are more readable and I have changed that.
I haven't added a JavaDoc, as I don't really know how the method behaves. If you want I can have a deeper look though.

**Comment by Siedlerchr** [comment]
@Ka0o0  Thanks for your contribution again! Could you please fix the checkstyle error? Then it's ready for merge

**Comment by Ka0o0** [comment]
@Siedlerchr I forgot to remove the unused import. It should pass the style check now.

**Pull request merged**